modules/core/src/main/resources/default.scala-steward.conf

```diff
@@ -0,0 +1,59 @@
+updates.ignore = [
+  // Ignore the next Scala 3 version until it is announced.
+  { groupId = "org.scala-lang", artifactId = "scala3-compiler",      version = { exact = "3.1.1" } },
+  { groupId = "org.scala-lang", artifactId = "scala3-library",       version = { exact = "3.1.1" } },
+  { groupId = "org.scala-lang", artifactId = "scala3-library_sjs1",  version = { exact = "3.1.1" } },
+
+  // Ignore the next Scala 2.13 version until it is announced.
+  { groupId = "org.scala-lang", artifactId = "scala-compiler", version = { exact = "2.13.8" } },
+  { groupId = "org.scala-lang", artifactId = "scala-library",  version = { exact = "2.13.8" } },
+  { groupId = "org.scala-lang", artifactId = "scala-reflect",  version = { exact = "2.13.8" } },
+  { groupId = "org.scala-lang", artifactId = "scalap",         version = { exact = "2.13.8" } },
+
+  // Ignore the next Scala 2.12 version until it is announced.
+  { groupId = "org.scala-lang", artifactId = "scala-compiler", version = { exact = "2.12.16" } },
+  { groupId = "org.scala-lang", artifactId = "scala-library",  version = { exact = "2.12.16" } },
+  { groupId = "org.scala-lang", artifactId = "scala-reflect",  version = { exact = "2.12.16" } },
+  { groupId = "org.scala-lang", artifactId = "scalap",         version = { exact = "2.12.16" } },
```

Comment:
@SethTisue FYI: Here are the version numbers that need to bumped once 2.12.16 / 2.13.8 is announced.